code/script.js

```diff
@@ -30,14 +45,181 @@ const showMessage = (message, sender) => {
+    <button id="rlsmBtn">Realism</button>
+  `
+
+  document
+    .getElementById('osBtn')
+    .addEventListener('click', () => nextQuestion('oldschool'))
+  document
+    .getElementById('bwBtn')
+    .addEventListener('click', () => nextQuestion('blackandgrey'))
+  document
+    .getElementById('rlsmBtn')
+    .addEventListener('click', () => nextQuestion('realism'))
+}
+
+//QUESTION 3
+
+const showDesigns = (type) => {
+  questionNumber = 3
+
+  botReply(
+    `Excellent choice, let's get you a ${type}-type tattoo. What design will that be?`
+  )
+
+  if (type === 'oldschool') {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>👇 Select a design...</option>
+        <option value="rose">Rose</option>
+        <option value="skull">Skull</option>
+        <option value="anchor">Anchor</option>
+      </select>
+    `
+  } else if (type === 'blackandgrey') {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>👇 Select a design...</option>
+        <option value="portrait">Portrait</option>
+        <option value="animal">Animal</option>
+        <option value="flowers">Flowers</option>
+      </select>
+    `
+  } else {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>👇 Select a design...</option>
+        <option value="Superhero/villain">Superhero/villain</option>
+        <option value="religious">Religious</option>
+        <option value="mythology">Mythology</option>
+      </select>
+    `
+  }
+  const select = document.getElementById('select')
+  select.addEventListener('change', () => nextQuestion(select.value))
+}
+
+//QUESTION 4
+
+const showPlacement = (design) => {
+  questionNumber = 4
+
+  botReply(`One ${design} coming up! Will that be on your arm or leg?`)
+
+  inputWrapper.innerHTML = `
+    <button id="arm">💪</button>
+    <button id="leg">🦵</button>
+  `
+
+  document
+    .getElementById('arm')
+    .addEventListener('click', () => nextQuestion('arm'))
+  document
+    .getElementById('leg')
+    .addEventListener('click', () => nextQuestion('leg'))
+}
+
+
+//QUESTION 5
+const showPrice = (placement) => {
+  questionNumber = 5
+
+  let price
+  if (placement === 'arm') {
+    price = '€150'
+  } else {
+    price = '€200'
+  }
+
+  botReply(
+    `We'll get your ${placement} decorated in no time. That'll be ${price}. Are you ready to confirm your appointment?`
+  )
+
+  inputWrapper.innerHTML = `
+    <button id="restart">NO</button>
+    <button id="confirm">YES</button>
+  `
+
+  document.getElementById('restart').addEventListener('click', () => {
+    location.reload()
+    return false
+  })
+  document
+    .getElementById('confirm')
+    .addEventListener('click', () => nextQuestion('Yes!'))
+}
+
+
+//QUESTION 2
+const thankYou = () => {
+  botReply(`Thank you for your booking!`)
+  inputWrapper.innerHTML = ``
+}
+
+
+
 // Set up your eventlisteners here
 
```

Comment:
We think the reason you are not able to enter name in first box is because the event listeners (line 215-221) are outside of the functions so the will activate, when we commented out these lines, the bot worked.